.vscode/settings.json

```diff
@@ -1,8 +1,10 @@
 {
-  "editor.rulers": [80],
+  "editor.rulers": [
```

Comment:
Can we make sure we don't include any unintentional settings.json changes (undo these and "Save without formatting")